hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java

```diff
@@ -207,7 +207,7 @@ private void readFileStream(final InputStream tempStream) {
                 eventBus.publish(this, new TargetTableEvent(TargetComponentEvent.BULK_UPLOAD_PROCESS_STARTED));
                 while ((line = reader.readLine()) != null) {
                     innerCounter++;
-                    readEachLine(line, innerCounter, totalFileSize);
+                    readEachLine(line, innerCounter, totalNumberOfLines);
```

Comment:
It makes sense to rename the variable to `totalNumberOfLines`, but than please rename the method parameter `totalFileSize` as well